client/blocks/reader-post-card/blocked.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import * as stats from 'reader/stats';
+import SiteBlockActions from 'lib/reader-site-blocks/actions';
+import Card from 'components/card';
+
+class PostBlocked extends React.PureComponent {
+
+	static propTypes = {
+		post: React.PropTypes.object,
+	};
+
+	unblock = () => {
+		analytics.mc.bumpStat( 'reader_actions', 'unblocked_blog' );
+		analytics.ga.recordEvent( 'reader_actions', 'Clicked Unblock Site' );
+		stats.recordTrack( 'calypso_reader_unblock_site', {
+			blog_id: this.props.post.site_ID,
+		} );
+		SiteBlockActions.unblock( this.props.post.site_ID );
+	}
+
+	render() {
+		const { post, translate } = this.props;
+
+		return (
+			<Card className="reader-post-card is-blocked">
+				<p className="reader-post-card__blocked-description">
+					{ translate( 'You have blocked all posts from %(site_name)s.', { args: { site_name: post.site_name } } ) }
+					<a onClick={ this.unblock } className="reader-post-card__blocked-undo">{ translate( 'Undo?' ) }</a>
```

Comment:
@obenland Hi there. I don't think we've interacted before. I'm Derek, the design lead on Reader. So I'm the "UX expert" you asked for. And my 2c is, punctuation in links is fine, so long as the whole sentence is a link. And in this case, it is.

Fwiw, @bluefuton has been working on Reader even longer than I have and is one of the most experienced people on Reader UI we have. So, while we always love a visit from @folletto, we don't need to wait on that to proceed here.